akkaclusterback/src/main/scala/com/boldradius/conductr/examples/AkkaClusterBackend.scala

```diff
@@ -50,15 +50,12 @@ object AkkaClusterBackend extends App with LazyLogging {
 
   logger.info(s"SEED NODES ${sys.props.get("akka.cluster.seed-nodes.0")}")
 
-
-  if( !Env.isRunByConductR ){
-    sys.props ++= List("akka.cluster.seed-nodes.0" -> "akka.tcp://AkkaConductRExamplesClusterSystem@127.0.0.1:8089")
-  }
-
   val system = ActorSystem("AkkaConductRExamplesClusterSystem", config)
 
-  system.actorOf(Props(classOf[AkkaClusterBackend]))
-
+  Cluster(system).registerOnMemberUp {
+    system.actorOf(Props(classOf[AkkaClusterBackend]))
+    StatusService.signalStartedOrExit()
+  }
```

Comment:
This is a cleaner method of bringing things online once the cluster is up. ConductR does something similar, and won't expose http endpoints etc. until the cluster has been established. That can prevent errors.